client/src/redux/sagas/apps.sagas.js

```diff
@@ -2,18 +2,18 @@ import { put, takeLatest, call, select } from 'redux-saga/effects';
 import 'cross-fetch';
 import { fetchUtil } from 'utils/fetchUtil';
 
-export async function fetchAppDefinitionUtil(appId) {
+export async function fetchAppDefinitionUtil(appId, appVersion) {
```

Comment:
Will this conflict with @nathanfranklin by chance?

Reply:
I think it could. 
I did base it off your apps branch to make sure that it's consistent. I removed the backend portion for the pr after testing that it works.
Here, it's required to [populate some of the job information](https://github.com/TACC/Core-Portal/blob/c2c20b0e777290fc7d43e000c7cb752d8ef0116a/client/src/redux/sagas/jobs.sagas.js#L121).

But I could remove it here!

Reply:
I have the same changes (appVersion added to  action payload and then same fetchAppDefinitionUtil signature changes) . so no need to remove here.

Reply:
It seems to work even without the changes (due to the try catch block in the job saga). I'll remove it in this pr to avoid confusion!